### PR TITLE
DON'T MERGE: Fix Docker configuration to work correctly with openidc-baseimage

### PR DESCRIPTION
**Pull request by coreone**
The `run.sh` script that comes with openidc-baseimage is pretty important, so this PR is to keep the `run.sh` in place while still allowing customizations via the `override.sh` script.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM broadinstitute/openidc-baseimage:1.8.5
+FROM broadinstitute/openidc-baseimage:1.8.6
```
Comment (dmohs) [inline Dockerfile:1]: Since we're not using any of the authentication bits, maybe `FROM httpd:2.4` is more appropriate?

**Comment by rushtong** [comment]
FYI, the build for this image is hanging for me and never gets past setting up certificates:
```
...
Creating config file /etc/php5/cli/php.ini with new version
php5_invoke pdo: already enabled for cli SAPI
php5_invoke opcache: already enabled for cli SAPI
php5_invoke json: already enabled for cli SAPI
Setting up libnss3-nssdb (2:3.19.2.1-0ubuntu0.14.04.1) ...
Setting up libnss3:amd64 (2:3.19.2.1-0ubuntu0.14.04.1) ...
Setting up openjdk-8-jre-headless:amd64 (8u72-b05-5~trusty1) ...
update-alternatives: using /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java to provide /usr/bin/java (java) in auto mode
update-alternatives: using /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/jjs to provide /usr/bin/jjs (jjs) in auto mode
update-alternatives: using /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/keytool to provide /usr/bin/keytool (keytool) in auto mode
update-alternatives: using /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/pack200 to provide /usr/bin/pack200 (pack200) in auto mode
update-alternatives: using /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/rmid to provide /usr/bin/rmid (rmid) in auto mode
update-alternatives: using /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/rmiregistry to provide /usr/bin/rmiregistry (rmiregistry) in auto mode
update-alternatives: using /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/unpack200 to provide /usr/bin/unpack200 (unpack200) in auto mode
update-alternatives: using /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/orbd to provide /usr/bin/orbd (orbd) in auto mode
update-alternatives: using /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/servertool to provide /usr/bin/servertool (servertool) in auto mode
update-alternatives: using /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/tnameserv to provide /usr/bin/tnameserv (tnameserv) in auto mode
update-alternatives: using /usr/lib/jvm/java-8-openjdk-amd64/jre/lib/jexec to provide /usr/bin/jexec (jexec) in auto mode
Setting up libatk-wrapper-java (0.30.4-4) ...
Setting up ca-certificates-java (20130815ubuntu1) ...
```

**Comment by davidangb** [comment]
@coreone is this ready to go? Before the holiday break I saw a note to hold off on this ...

**Comment by coreone** [comment]
Yes, this should be good now.  We were working out some things with openidc-baseimage, but they have been sorted, so it should be good to go.

**Comment by davidangb** [comment]
Finally getting back to looking at this. I was able to build the image, but after starting it (via uiorch-compose.yaml in firecloud-environment), I do not see FireCloud. Apache says it is started, but it's not serving any content. Did this actually work for you? I haven't tried to debug it. @coreone

**Comment by dmohs** [comment]
I suggest we close this PR and instead create a PR to move to a standard Apache or Nginx proxy.

**Comment by coreone** [comment]
@dmohs that makes sense.  We have a straight "enforce HTTPS" apache container currently that we use on our Jenkins hosts and other things.  Maybe we can do some updates on that to get this working for you in a more streamlined fashion.

**Comment by dmohs** [comment]
@coreone We might be able to use that. Right now, we use HTTP for local development because Figwheel connects over an HTTP websocket to port 3449 and Chrome will not allow that if the page is HTTPS. Updating the Figwheel code to support HTTPS would be a long-term goal.

However, we might be able to proxy the websocket port so we can use HTTPS. It's something I've been wanting to look into.

**Comment by dmohs** [comment]
I'm pairing with @coreone to redo this using the updated configuration architecture. Closing.

**Pull request closed without merging**